test-cypress/support/commands.js

```diff
@@ -79,13 +87,16 @@ Cypress.Commands.add('selectRDFLinkButton', () => {
 
 
 Cypress.Commands.add('selectRepo', (repoId) => {
-    cy.get('#repositorySelectDropdown')
-        .click({force: true})
-        .then(() => {
-            cy.get('.dropdown-item', {timeout: 1000})
-                .contains(repoId)
-                .click({force: true});
-        });
+    cy.get('#repositorySelectDropdown', {timeout: 30000}).should('be.visible');
```

Comment:
Is the timeout needed? Can we enforce it globally or it will slow down the suite?

Reply:
Some times the repo dropdown is not rendered at all and I have no idea why, that's why I've added this particular wait.

On the global timeout note: default timeout is 4 seconds, which should be enough for most UI operations (in the workbench) and I don't want to change it globally, only in specific cases.

Reply:
Raised a JIRA (GDB-3768) to check if the problem is in the workbench code.